docs/getting_started.md

```
@@ -14,38 +14,45 @@ This tutorial will provide a brief overview of:
+Select the **landfirevegmodels** package and choose either the **CONUS and HI Reference Conditions** for continental U.S. and Hawaii models or the **Alaska Reference Conditions** for models of Alaska.
 
 Enter a **Filename** or keep the default, select a **Folder** for your new library, and click **OK**. A download window should appear and your library will be created from an online template.
 
 <br>
 
 ## Configuring the Library
 
-Right-click on the Project datafeed (*i.e.*, **Definitions**) and select **Properties**. Navigate to the **Landfire** tab, where you can specify details about the quantity, type, and composition of the vegetation for your model.
+Right-click on the Project datafeed (*i.e.*, **BpSModels**) and select **Open** (or **double-click** on BpSModels). Navigate to the **LANDFIRE vegetation models** tab, where you can specify details about the quantity, type, and composition of the vegetation for your model.
+
+<img align="middle" style="padding: 1px" width="500" src="assets/images/image-4">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)